lib/forge_sdk/conn/rpc_conn.ex

```diff
@@ -130,6 +136,23 @@ defmodule ForgeSdk.RpcConn do
     {:noreply, %{state | conn: %{conn | chain_id: chain_id, decimal: decimal}, config: config}}
   end
 
+  def handle_cast(:update_gas, %{conn: conn} = state) do
```

Comment:
So you do not reconnect but keep updating the gas for the sdk？

Reply:
Yes. The reason is: when forge starts, there will be one grpc connection established between ForgeSdk client and forge. We need to have a grpc connection in order to get the forge_state.
However, the request of update gas will only issue **once** when call ForgeSdk.connect. If we disconnect the grpc connect, then when reconnected, there's no one issuing the update_gas request. So the solution here is if someone issue a update_gas request, then it will try to update gas every 1s, till forge_state is in state db.